packages/rocketchat-slashcommand-tableflip/tableflip.js

```diff
@@ -0,0 +1,20 @@
+/*
+* Tableflip is a named function that will replace /Tableflip commands
+* @param {Object} message - The message object
+*/
+
+
+function Tableflip(command, params, item) {
+	if (command === 'Tableflip') {
+		var msg;
+
+		msg = item;
+		msg.msg = params + ' ¯(╯°□°）╯︵ ┻━┻';
+		Meteor.call('sendMessage', msg);
+	}
+}
+
+RocketChat.slashCommands.add('Tableflip', Shrug, {
+	description: 'Slash_Tableflip_Description',
```

Comment:
You'll need to actually add the i18n files that include the description like done with shrug